### PR TITLE
Milin edit documentation python threads example

### PR DESCRIPTION
**Pull request by milin-fr**
### Type of Change

- Editing an existing entry (fixing a typo, bug, issues, etc)

### Checklist

- [x] All writings are my own.
- [x] My entry follows the Codecademy Docs style guide.
- [x] My changes generate no new warnings.
- [x] I have performed a self-review of my own writing and code.
- [x] I have checked my entry and corrected any misspellings.
- [x] I have made corresponding changes to the documentation if needed.
- [x] I have confirmed my changes are not being pushed from my forked `main` branch.
- [x] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [ ] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above). - There were no open issues corresponding to this pull request.

### CODE REVIEW

**Review by SSwiniarski — CHANGES_REQUESTED** [review]
@milin-fr, I did a first review and left a few comments, generally to do with the spacing of the code example. (We like indents to be two spaces rather than four.)

**Comment by milin-fr** [comment]
Thank you for the review! I accepted the changes (and hope you didn’t get spammed with email for each action : )

**Review by milin-fr — COMMENTED** [review]
Thank you for the review! I accepted suggested changes and will keep in mind 2 spaces for future commits.

**Review by SSwiniarski — APPROVED** [review]
@milin-fr Looks good for a second review.

**Comment by Dusch4593** [comment]
Hey @milin-fr I'll do the next review for this PR! 😄

File: content/python/concepts/threading/threading.md [hunk 1]
```diff
@@ -25,3 +25,44 @@ import threading
...
+    threads.append(individual_thread)
+    individual_thread.start()
+  # at this point threads are running independently from the main flow of application and each other
+  print("Main flow of application")
+  for individual_thread in threads:
+    individual_thread.join()
+  # joining threads insures that all threads finished their task before moving further in the main flow of application
+  print("All threads are done")
+
+run_threads()
```
Comment (Dusch4593) [inline content/python/concepts/threading/threading.md:55]:
```suggestion
def run_threads():
  threads = []
  
  for thread_index in range(5):
    individual_thread = threading.Thread(target=slow_function, args=(thread_index,))
    threads.append(individual_thread)
    individual_thread.start()

  # at this point threads are running independently from the main flow of application and each other
  print("Main flow of application")
  
  # This ensures that all threads finish before the main flow of application continues
  for individual_thread in threads:
    individual_thread.join()
  
  print("All threads are done")

run_threads()
```

File: content/python/concepts/threading/threading.md [hunk 2]
```diff
@@ -25,3 +25,44 @@ import threading
...
+```
+
+This results in output like the following
```
Comment (Dusch4593) [inline content/python/concepts/threading/threading.md:58]:
```suggestion
This results in output like the following:
```

**Review by Dusch4593 — CHANGES_REQUESTED** [review]
@milin-fr Left you some comments/suggestions to check out and implement. 😄 Excellent example!

**Comment by milin-fr** [comment]
Thank you for the review !

**Review by Dusch4593 — APPROVED** [review]
@milin-fr LGTM! 👍🏻

Made a super small edit in 2ae8f4ec5d3153caf2602e5d21af7bbbc6980f07 but looks good for merge, otherwise! 😄

**Comment by Dusch4593** [comment]
Congrats and thank you so much for contributing to Codecademy Docs, @milin-fr! 🎊 🎉 🥳

The changes should be live: https://www.codecademy.com/resources/docs/python/threading

Currently, your Codecademy profile is anonymous on the entry page. To make it visible, you'll need to add to your GitHub username to your Codecademy profile, log out of Codecademy, and then log back in through your GitHub account.

Looking forward to your next Docs contribution! 😄

**Pull request merged**